.github/workflows/test.yml

```diff
@@ -47,5 +47,5 @@ jobs:
       - name: Lint
         uses: golangci/golangci-lint-action@v2.5.2
         with:
-          version: v1.47.3
+          version: v1.52.2
```

Comment:
I noticed master is `version: v1.51.2`, do we need to update the master as well?
https://github.com/tikv/client-go/blob/master/.github/workflows/test.yml#L50

Reply:
I think we can .